drivers/storage/portworx/util/feature.go

```diff
@@ -10,6 +10,8 @@ type Feature string
 const (
 	// FeatureCSI is used to indicate CSI feature
 	FeatureCSI Feature = "CSI"
+	// FeatureBottleRocketAMI is used to activate BottleRocket AMI tweaks
```

Comment:
Instead of feature flag, can we put this as an annotation here - https://github.com/libopenstorage/operator/blob/master/drivers/storage/portworx/util/util.go#L58?

Feature flags was reserved mostly for portworx features like CSI, may be dm-thin, etc.
Also, eventually we want to remove these annotations and auto-detect most things about the environment.